haystack/components/converters/html.py

```diff
@@ -28,6 +28,27 @@ class HTMLToDocument:
 
     """
 
+    def __init__(
+        self,
+        extractor_type: Literal[
+            "DefaultExtractor",
+            "ArticleExtractor",
+            "ArticleSentencesExtractor",
+            "LargestContentExtractor",
+            "CanolaExtractor",
+            "KeepEverythingExtractor",
+            "NumWordsRulesExtractor",
+        ] = "DefaultExtractor",
```

Comment:
Why not use the class directly? 🤔

Reply:
I adopted the [same approach of the `InMemoryDocumentStore`](https://github.com/deepset-ai/haystack/blame/dcf37c517359b2235b5524bde68ca662bdcf9c8f/haystack/document_stores/in_memory/document_store.py#L37), to avoid serialization issues.

Do you have any better implementation ideas?

Reply:
Ok then.

I created issue #6588 for the serialization by the way since it's missing completely for this Component.